src/main/webapp/creatives/maps/simid_maps.js

```diff
@@ -5,4 +5,25 @@ class SimidMapCreative extends BaseSimidCreative {
   constructor() {
     super();
   }
+  /** @override */
+  onStart(eventData){
+    super.onStart(eventData);
+    this.adParams = JSON.parse(this.creativeData.adParameters);
```

Comment:
Instead of saving it directly to adParams, probably best to parse out each param individually and save it.
You may also want to consider using default values incase adParams is malformed or missing data

Reply:
was planning on adding DEFAULT values today

Reply:
Agreed